doc/openapi-user-guide.md

```diff
@@ -92,6 +93,77 @@ spec:
 
 [OpenAPI CRD Reference](openapi-reference.md) for more info.
 
+### OpenID Connect and OAuth2 example
+
+3scale requires some additional information that are not included 
+in the OpenAPI spec, therefore those data need to be provided in the OpenAPI CR. Specifically:
+- OpenID Connect Issuer Type, which will default to rest but can be overridden from the OpenAPI CR
+- OpenID Connect Issuer Endpoint Referense (Secret). 3scale requires that the issuer URL must include a client secret.
+- Flows object. When the sec scheme is oauth2, the flows are provided by the OpenAPI doc. However, for openIdConnect security scheme, the OpenAPI doc does not provide the flows. In that case, the OpenAPI CR can provide those.
+
+#### OIDC Issuer Secret
+
```

Comment:
Think you need to call out that your have previously setup keycloak realm and client and are using it as your example . Other wise know way to know where the issuerEndpoint comes from.

Reply:
and how the issuerEndpoint is constructed ``` https://<client_id>:<client_secret>@<rhsso_host>:<rhsso_port>/auth/realms/<realm_name>```

Reply:
Thank you for comment. Hope it's a bit better now. Done.